### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by mugishasam123**
## Bookstore

-  Add structure of the app
- Add 2 links to the navbar
- Add form and categories page that displays text
- The building blocks of my app are set as re-usable components.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        "@testing-library/jest-dom": "^5.16.2",
+        "@testing-library/react": "^12.1.2",
+        "@testing-library/user-event": "^13.5.0",
+        "big.js": "^6.1.1",
+        "gh-pages": "^3.2.3",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
+        "react-router-dom": "^6.2.1",
+        "react-scripts": "5.0.0",
+        "web-vitals": "^2.1.4"
```
Comment (mikethreels) [inline package.json]: - [ ] according to the requirements you should add React Redux to the dependencies `npm install react-redux`.

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <select name="category" id="category">
+          <option value="history">History</option>
+          <option value="english">English</option>
+          <option value="economy">Economy</option>
+          <option value="literature">Literature</option>
+        </select>
+      </form>
+    </div>
+  </>
+);
```
Comment (mikethreels) [inline src/components/Books.js]: - [ ] In this component right now you are just displaying HTML/JSX code it doesn't really function as intended. the books component should render the `Book` and `CreateBook` components.
- the `Book` component takes the book data as a prop and renders the title and the author and has a button for removing books
- the `CreateBook` component renders a form with inputs for the title and the author of the book and has a button to submit
Both the button for removing books and for submitting the form should be placeholder buttons right now they don't have to do anything.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Samuel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉 

## Hi @mugishasam123, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**